tensorflow/c/eager/dlpack.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+
+#ifndef TENSORFLOW_C_DLPACK_H_
+#define TENSORFLOW_C_DLPACK_H_
+
+#include "tensorflow/c/eager/c_api.h"
+#include "tensorflow/core/framework/tensor.h"
+
+namespace tensorflow {
+
+TF_CAPI_EXPORT extern const char* const kDlTensorCapsuleName = "dltensor";
```

Comment:
Does this actually work?  I'd have expected this cause duplicate definitions of `kDlTensorCapsuleName`.

Reply:
Removed